src/L1/rollup/IScrollChain.sol

```diff
@@ -41,6 +41,14 @@ interface IScrollChain {
     /// @param newMaxNumTxInChunk The new value of `maxNumTxInChunk`.
     event UpdateMaxNumTxInChunk(uint256 oldMaxNumTxInChunk, uint256 newMaxNumTxInChunk);
 
+    /// @notice Emitted when enter enforced batch mode.
+    /// @param lastCommittedBatchIndex The index of last committed batch.
+    event EnterEnforcedBatchMode(uint256 lastCommittedBatchIndex);
+
+    /// @notice Emitted when exit enforced batch mode.
+    /// @param lastCommittedBatchIndex The index of last committed batch.
+    event ExitEnforcedBatchMode(uint256 lastCommittedBatchIndex);
```

Comment:
I'd probably just use a single event, but both ways are fine.

```suggestion
    /// @notice Emitted when we enter or exit enforced batch mode.
    /// @param enabled True if we are entering enforced batch mode, false otherwise.
    /// @param lastCommittedBatchIndex The index of last committed batch.
    event UpdateEnforcedBatchMode(bool enabled, uint256 lastCommittedBatchIndex);
```